acceptance/tests/resource/service/windows.rb

```diff
@@ -37,23 +37,12 @@ def service_manifest(name, params)
   # 10 seconds, having the service sleep for 20
   # will supply the failure scenario on a failed
   # startup
-  mock_service_startfail = {
-    :name => "mock_service_startfail",
-    :start_sleep => 60,
+  mock_service_long_start_stop = {
+    :name => "mock_service_long_start_stop",
+    :start_sleep => 40,
```

Comment:
Assuming these wait times don't conflict with the max 10 seconds thing?

Reply:
Whoops, that comment is outdated. No these wait times won't conflict with the wait hint described in that comment, will update the comment.